files/es/web/javascript/reference/operators/remainder/index.md

```diff
@@ -25,33 +25,33 @@ Operador: var1 % var2
 ### Resto con dividendo positivo
 
 ```js
- 12 % 5  //  2
- 1 % -2 //  1
- 1 % 2  //  1
- 2 % 3  //  2
-5.5 % 2 // 1.5
+12 % 5; //  2
+1 % -2; //  1
+1 % 2; //  1
+2 % 3; //  2
+5.5 % 2; // 1.5
 ```
 
 ### Resto con dividendo negativo
 
 ```js
--12 % 5 // -2
--1 % 2  // -1
--4 % 2  // -0
+(-12 % 5) - // -2
+  (1 % 2) - // -1
+  (4 % 2); // -0
 ```
```

Comment:
Just a note, that are for the example

````
```js-nolint
-12 % 5 // -2
-1 % 2  // -1
-4 % 2  // -0
```
````